scripts/slackbot.js

```diff
@@ -42,6 +22,65 @@ try {
   console.error(`Error:`, error.message);
 }
 
+async function slackbot(botToken) {
+  const greeting = getGreeting(updatesArray.length);
+
+  const sortedUpdates = updatesArray.reduce(
+    (updates, component) => {
+      if (component.version.endsWith('.0.0')) updates.major.push(component);
+      else if (component.version.endsWith('.0')) updates.minor.push(component);
+      else updates.patch.push(component);
+      return updates;
+    },
+    {
+      major: [],
+      minor: [],
+      patch: [],
+    },
+  );
+
+  const majorUpdatesString = (
+    await Promise.all(sortedUpdates.major.map(parseMajorChangeString))
+  ).join('\n');
+
+  const minorUpdatesString = sortedUpdates.minor
+    .map(component => {
+      const { fullName, changelogUrl } = generateOutputStrings(component);
+      return `*<${changelogUrl} | ${fullName}>*`;
+    })
+    .join('\n');
+
+  const patchUpdatesString = sortedUpdates.patch
+    .map(component => {
+      const { fullName, changelogUrl } = generateOutputStrings(component);
+      return `<${changelogUrl} | ${fullName}>`;
+    })
+    .join(', ');
+
+  let updatesString = '';
+  if (majorUpdatesString.length > 0)
+    updatesString += `*Major Changes*\n${majorUpdatesString}`;
+  if (minorUpdatesString.length > 0)
+    updatesString += `\n\n*Minor Changes*\n${minorUpdatesString}`;
+  if (patchUpdatesString.length > 0)
+    updatesString += `\n\n*Patche Changes*\n${patchUpdatesString}`;
```

Comment:
good ol' `Patchè` change lol

Reply:
Only if it comes from the `Patchè` region of France, otherwise it's just a sparkling bug fix